src/error_strings.c

```diff
@@ -54,26 +53,30 @@ gchar *failed_string = NULL;
 #define INIT_ERRMSG(errvar, errtxt) \
 	{ \
 		errvar = g_strdup(errtxt); \
-		if (!no_indata_expected_error) { \
-			printf("%s\n", ALLOCATE_ERROR_STRING); \
+		if (!errvar) { \
+			printf("%s: \"%s\"\n", ALLOCATE_ERROR_STRING, errtxt); \
 			return -1; \
 		} \
 	}
 int init_script_error_messages()
 {
-	INIT_ERRMSG(no_indata_expected_error,                   _("No indata expected"));
-	INIT_ERRMSG(one_indata_expected_error,                  _("One indata expected"));
-	INIT_ERRMSG(two_indata_expected_error,                  _("Two indata expected"));
-	INIT_ERRMSG(four_indata_expected_error,                 _("Four indata expected"));
+	INIT_ERRMSG(num_indata_expected_errors[0],              _("No indata expected"));
+	INIT_ERRMSG(num_indata_expected_errors[1],              _("One indata expected"));
+	INIT_ERRMSG(num_indata_expected_errors[2],              _("Two indata expected"));
+	INIT_ERRMSG(num_indata_expected_errors[3],              _("Three indata expected"));
+	INIT_ERRMSG(num_indata_expected_errors[4],              _("Four indata expected"));
+
+	INIT_ERRMSG(n_or_m_indata_expected_error,				_("%d or %d indata expected"));
+	INIT_ERRMSG(n_to_m_indata_expected_error,				_("%d to %d indata expected"));
```

Comment:
Better to use spaces after initial depth indentation. Assumptions about tab widths won't matter that way.

Reply:
Thanks. I'll try to convince my editor to do this by default.